package.json

```diff
@@ -72,15 +72,15 @@
     "express": "^4.16.4",
     "express-session": "^1.15.6",
     "express-validator": "^6.0.0",
-    "govuk-frontend": "^2.12.0",
+    "govuk-frontend": "^3.2.0",
     "helmet": "^3.15.1",
```

Comment:
Is it worth upgrading helmet at the same time? This is the change in that Snyk pr: 
"helmet": "^3.21.1",

Reply:
I would like to keep them separate so we can easily roll back the upgrade if things go wrong. The `node-sass` update is related to the upgrade.